CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 10.5
+
+* Introduce `org.sonar.api.issue.IssueStatus` to simplify `status` and `resolution` on issues.
+* Deprecate `org.sonar.api.ce.measure.Issue.status()` and `org.sonar.api.ce.measure.Issue.resolution()`. Use `org.sonar.api.ce.measure.Issue.issueStatus()` method instead.
+* Deprecate `org.sonar.api.issue.Issue.status()` and `org.sonar.api.issue.Issue.resolution()`. No replacement.
```

Comment:
Isn't the new `issueStatus` the replacement?

Reply:
For `org.sonar.api.issue.Issue` specifically, we didn't introduced the issueStatus() method, as this class is only use internally by the product, and we might want to deprecate it and remove it at some point. we did the same with impacts and clean code attribute and did not add it to this class